### PR TITLE
PLATUI-2818: Service teams can filter out urls/page paths that they do not own from their a11y reports

### PR DESCRIPTION
**Pull request by kyle-bowden**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,53 @@
...
-### Scalafmt
-
-Check all project files are formatted as expected as follows:
-
-```bash
-sbt scalafmtCheckAll scalafmtCheck
+### Getting started
+* Ensure your UI test project is using [ui-test-runner](https://github.com/hmrc/ui-test-runner), eg. in your dependencies
+```scala
+    "uk.gov.hmrc" %% "ui-test-runner" % "0.17.0" % Test
```
[the remark this answers is not in the document]
Comment (ellamdav) [reply README.md:10]: Yeah, fair enough - we were just trying to be really explicit, and the "eg." was supposed to imply checking for the latest version, but probably gives teams scope to shoot themselves in the foot.  Maybe link to https://github.com/hmrc/ui-test-runner/releases instead?

File: README.md [hunk 2]
```diff
@@ -1,29 +1,53 @@
...
+* Add this plugin to your project, eg.
+```scala
+    addSbtPlugin("uk.gov.hmrc" % "sbt-test-report" % "0.22.0")
 ```
-
-Format all project files as follows:
-
-```bash
-sbt scalafmtAll
+* Update your build job to run `testReport` *after* your UI tests, eg.
```
[the remark this answers is not in the document]
Comment (ellamdav) [reply README.md]: Agreed - we should update [DEFINING-JOBS.md](https://github.com/hmrc/build-jobs/blob/main/DEFINING-JOBS.md) with how to integrate with the a11y report and then just link to that here

File: README.md [hunk 3]
```diff
@@ -1,29 +1,53 @@
...
+```groovy
+    .withTests("... acceptance:test testReport")
+```
+* and also add publishers to save the report and other artefacts with your completed build
+```groovy
+    .withPublisher(accessibilityAssessmentArtifactsPublisher())
```
[the remark this answers is not in the document]
Comment (ellamdav) [reply README.md:22]: Sorry, my bad for copying from one of our PlatUI jobs, which aren't yet following the recommended approach... Will remove this bit.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ellamdav) [reply README.md]: We've just copied an example shell script from the migration guide here - didn't feel like that belonged in build-jobs documentation, and didn't want to link to the migration guide in confluence from here

**Review by vivrichards600 — APPROVED** [review]
Myself and Tom have had a review. Changes look good.

**Pull request merged**